File: ClinicSystem/src/com/example/clinicsystem/models/enums/DentistryDepartment.java
package com.example.clinicsystem.models.enums;

public enum DentistryDepartment {
    DIAGNOSIS("Diagnosis", 1),
    OPERATIVE("Operative", 2),
    ENDODONTICS("Endodontics", 3),
    REMOVABLE_PROSTH("Removable Prosth", 4),
    PERIODONTICS("Periodontics", 5),
    SURGERY("Surgery", 6),
    CROWN_AND_BRIDGE("Crown and Bridge", 7),
    PEDODONTICS("Pedodontics", 8),
    POST_GRADUATE("Post Graduate", 9);

    private final String name;
    private final int id;

    DentistryDepartment(String name, int id) {
        this.name = name;
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public int getId() {
        return id;
    }

    public static DentistryDepartment getConstant(String name) {
        for (DentistryDepartment dentistryDepartment : DentistryDepartment.values()) {
            if (dentistryDepartment.getName().equals(name)) {
                return dentistryDepartment;
            }
        }

        return null;
    }

    public static DentistryDepartment getConstant(int id) {
        for (DentistryDepartment dentistryDepartment : DentistryDepartment.values()) {
            if (dentistryDepartment.getId() == id) {
                return dentistryDepartment;
            }
        }

        return null;
    }
}

File: clinic/DentistryDepartment.php
<?php
class DentistryDepartment{
	private const $DIAGNOSIS		= array("Diagnosis", 1);
    private const $OPERATIVE		= array("Operative", 2);
    private const $ENDODONTICS		= array("Endodontics", 3);
    private const $REMOVABLE_PROSTH	= array("Removable Prosth", 4);
    private const $PERIODONTICS		= array("Periodontics", 5);
    private const $SURGERY			= array("Surgery", 6);
    private const $CROWN_AND_BRIDGE = array("Crown and Bridge", 7);
    private const $PEDODONTICS	    = array["Pedodontics", 8);
    private const $POST_GRADUATE    = array["Post Graduate", 9);
	private const $INSTANCE;
	
	private __construct() { 
		$this->INSTANCE = array();
		array_push($this->INSTANCE, $this->DIAGNOSIS);
		array_push($this->INSTANCE, $this->OPERATIVE);	
		array_push($this->INSTANCE, $this->ENDODONTICS);
		array_push($this->INSTANCE, $this->REMOVABLE_PROSTH);
		array_push($this->INSTANCE, $this->PERIODONTICS);
		array_push($this->INSTANCE, $this->SURGERY);
		array_push($this->INSTANCE, $this->CROWN_AND_BRIDGE);
		array_push($this->INSTANCE, $this->PEDODONTICS);
		array_push($this->INSTANCE, $this->POST_GRADUATE);		
	}
	
	public static function getINSTANCE() {
		if ($this->INSTANCE == null) {
			$this->INSTANCE = new DentistryDepartment();
		}
		
		return $this->INSTANCE;
	}
	
	
}
?>